git/helpers.sh

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
Really need a shebang here.

Reply:
I believe my script is completely portable between shells (and if it is not, we can make it so).

Reply:
Cool, nevermind then.